src/react-apps/ux-editor/src/containers/Container.tsx

```diff
@@ -48,7 +48,8 @@ export class ContainerComponent extends React.Component<IContainerProps> {
     );
 
     ConditionalRenderingActionDispatcher.checkIfConditionalRulesShouldRun();
-    ApiActionDispatchers.checkIfApiShouldFetch(id, dataModelElement, callbackValue);
+    RuleConnectionActionDispatchers.checkIfRuleShouldRun(id, dataModelElement, callbackValue, this.props.dataModelGroup, this.props.index);
+    ApiActionDispatchers.checkIfApiShouldFetch(id, dataModelElement, callbackValue, this.props.dataModelGroup, this.props.index);
     RuleConnectionActionDispatchers.checkIfRuleShouldRun(id, dataModelElement, callbackValue);
```

Comment:
Should this line be removed?
```suggestion
```

Reply:
Actually I should remove the "duplicate" call
```suggestion
    RuleConnectionActionDispatchers.checkIfRuleShouldRun(id, dataModelElement, callbackValue);
```
As the beheviour is the same when passing along the dataModelGroup and index. Think I just missed it when merging.